### PR TITLE
fix documentation for osx build

### LINKED ISSUE #101: OSX Dev setup
Does anyone have tried to build the GUI on OSX? 
If so, please advice what you have done and what am I missing.

It seems to me that all PRs are failing for OSX10.11 and 10.12 due to some issue in OSX environment. 

I'm trying to build it myself and also facing the issues.

The first issue was in Qt that needed to change a string from `xcrun` to `xcodebuild`(source: [link](https://stackoverflow.com/questions/33728905/qt-creator-project-error-xcode-not-set-up-properly-you-may-need-to-confirm-t) )

The second issue was that in `libwalletqt` folder it was expecting to have [wallet](https://github.com/monero-project/monero/tree/master/src/wallet) folder from actual monero project that has a file  `wallet2_api.h` I symlinked the folder and build started running, but

The third issue that I see now is 
```
d: warning: directory not found for option '-L/YOUR_PATH/monero-core/monero/lib'
ld: library not found for -lwallet_merged
```
Let me know if someone is using OSX to build monero-core

**UPDATE:**
I was pointed out that I'd need run `./get_libwallet_api.sh` first and then run `build.sh`
which solves some problems.
Before that, you also need to have openssl headers installed

**Update**: 
All is working, so I'll update readme.md for osx setup to
1)`brew install openssl`
2)add `-L/usr/local/opt/openssl/lib \` in here
https://github.com/monero-project/monero-core/blob/master/monero-core.pro#L133-L134
3)run `./get_libwallet_api.sh`
4)run `./build.sh`

### PR DESCRIPTION
**Pull request by rstormsf**
Fixes #101

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -135,7 +137,11 @@ TODO
   git clone https://github.com/monero-project/monero-core.git
   
   cd monero-core
+  ```
+6.1 add ```-L/usr/local/opt/openssl/lib \``` in here ./monero-core.pro#L133-L134
```
Comment (Jaqueeee) [inline README.md]: Please add this to monero-core.pro instead.

**Comment by fluffypony** [comment]
Waiting on suggested changes...

**Comment by maitscha** [comment]
Instead I propose PR #138.

This adds the library path for the openssl and boost library, installed with `brew`.

**Comment by fluffypony** [comment]
Closing in favour of #138

**Pull request closed without merging**